redash/cli/users.py

```diff
@@ -200,11 +202,15 @@ def delete(email, organization=None):
     """
     Delete user EMAIL.
     """
+    deleted_users_query = sql_delete(models.User).where(models.User.email == email)
     if organization:
         org = models.Organization.get_by_slug(organization)
-        deleted_count = models.User.query.filter(models.User.email == email, models.User.org == org.id).delete()
-    else:
-        deleted_count = models.User.query.filter(models.User.email == email).delete(synchronize_session=False)
+        deleted_users_query = deleted_users_query.where(models.User.org == org)
+    deleted_count = len(
```

Comment:
It should not use `len` for getting deleted count. Please check:
https://docs.sqlalchemy.org/en/20/tutorial/data_update.html#getting-affected-row-count-from-update-delete

Reply:
updated

Reply:
Thanks for your update. Confirmed.